docs/integrations.rst

```diff
@@ -213,6 +213,16 @@ PHP Symfony Security Checker
 ----------------------------
 Import results from the PHP Symfony Security Checker.
 
+Probely
+-------
+Synchronize Probely Plus findings with DefectDojo.
+
+To setup this integration set the DefectDojo URL and API key on the Integrations page on Probely. Then, select which Product, Engagement, and, optionally, the Test you want to synchronize to. The API key needs to belong to a staff user.
+
+Works with DefectDojo 1.5.x and 1.6.x. Probely also supports non-public DefectDojo instances; please reach out to us at https://probely.com/ for help.

```

Comment:
We just released 1.7.0 by the way.

Reply:
So your other comment somewhere else. IIRC, the re-import function was untouched, so should be fine.

Reply:
I would avoid this link here, we should not act as a trampoline to your site ;-) The help link below is ok.

Reply:
done!